server/constants/requestMethod.ts

```diff
@@ -0,0 +1,8 @@
+const requestMethods: Record<string, string> = {
```

Comment:
Now we've got the API definitions in, the `Record<string,string>` could be tied down.